scrapy/http/request/__init__.py

```diff
@@ -103,3 +104,14 @@ def replace(self, *args, **kwargs):
             kwargs.setdefault(x, getattr(self, x))
         cls = kwargs.pop('cls', self.__class__)
         return cls(*args, **kwargs)
+
+    @classmethod
+    def from_curl(cls, curl_command, **kwargs):
+        """Create a Request object from a string containing a `curl <https://curl.haxx.se/>`_ command. It populates the
+        HTTP method, the URL, the headers, the cookies and the body. It accepts the same arguments as the
+        :class:`Request` class. Note that all the arguments provided take preference and will override the values of
+        the same arguments contained in the curl command.
```

Comment:
Could you please limit line length? We're not super-strict about it, but unless there is a reason, we try to follow e.g. 79 columns. For example, reviewing this diff on github requires horizontal scrolling :)